data/saas/dataset/twilio_dataset.yml

```diff
@@ -0,0 +1,249 @@
+            fidesops_meta:
+              data_type: string[]
+          - name: participant_sid
+            data_categories: [system.operations]
+            fidesops_meta:
+              data_type: string
+          - name: conversation_state
+            data_categories: [system.operations]
+            fidesops_meta:
+              data_type: string
+          - name: chat_service_sid
+            data_categories: [system.operations]
+            fidesops_meta:
+              data_type: string
+          - name: links
+            fields:
+              - name: conversation
+                data_categories: [system.operations]
+                fidesops_meta:
+                  data_type: string
+              - name: participant
+                data_categories: [system.operations]
+                fidesops_meta:
+                  data_type: string
+      - name: conversation_messages
+        fields:
+          - name: body
+            data_categories: [system.operations]
+            fidesops_meta:
+              data_type: string
+          - name: index
+            data_categories: [system.operations]
+            fidesops_meta:
+              data_type: string
+          - name: author
+            data_categories: [user.name]
+            fidesops_meta:
+              data_type: string
+          - name: date_updated
+            data_categories: [system.operations]
+            fidesops_meta:
+              data_type: string
+          - name: media
+            data_categories: [system.operations]
+            fidesops_meta:
+              data_type: string
+          - name: participant_sid
+            data_categories: [system.operations]
+            fidesops_meta:
+              data_type: string
+          - name: conversation_sid
+            data_categories: [system.operations]
+            fidesops_meta:
+              data_type: string
+          - name: account_sid
+            data_categories: [system.operations]
+            fidesops_meta:
+              data_type: string
+          - name: delivery
+            data_categories: [system.operations]
+            fidesops_meta:
+              data_type: string
+          - name: url
+            data_categories: [system.operations]
+            fidesops_meta:
+              data_type: string
+          - name: date_created
+            data_categories: [system.operations]
+            fidesops_meta:
+              data_type: string
+          - name: sid
+            data_categories: [system.operations]
+            fidesops_meta:
+              data_type: string
+          - name: attributes
+            data_categories: [system.operations]
+            fidesops_meta:
+              data_type: string[]
+          - name: links
+            fields:
+              - name: delivery_receipts
+                data_categories: [system.operations]
+                fidesops_meta:
+                  data_type: string
+      - name: conversation_participants
```

Comment:
Same as above, are there any fields we can flag under the user data category?

Reply:
As discussed in our conversation, this only has optional PII which we can't implement on our account. Current data doesn't have any field that we can flag under user data category.